keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -884,6 +991,24 @@ Describe the metrics themselves and the reasons why they weren't added (e.g., co
 implementation difficulties, etc.).
 -->
 
+Potentially, kube-scheduler could implement a rule to avoid scheduling a pod with `supplementalGroupsPolicy: Strict`
```

Comment:
I think I am OK with this; if they use `Strict` without targeting to nodes that can support it, it is "user error".